src/imp_std.rs

```diff
@@ -68,7 +68,9 @@ impl<T> OnceCell<T> {
         initialize_or_wait(
             &self.queue,
             Some(&mut || {
-                let f = unsafe { f.take().unwrap_unchecked() };
```

Comment:
The _purpose_ of `unwrap_unchecked` is to remove, at compile time, the possibility of the runtime panic, to make sure that the generated code doesn't contain panic invocation. In other words, this uses `unwrap_unchecked` specifically to avoid `.unwrap`, `unreacheable!()`, `panic!()` and frieds, to guarantee smaller codegen.